src/lib/docker-utils.ts

```diff
@@ -45,7 +45,7 @@ const DELTA_TOKEN_TIMEOUT = 10 * 60 * 1000;
 export class DockerUtils extends DockerToolbelt {
 	public dockerProgress: DockerProgress;
 
-	public constructor(opts: Dockerode.DockerOptions) {
+	public constructor(opts?: Dockerode.DockerOptions) {
```

Comment:
This is actually optional and the application-manager doesn't pass any options in